docs/guides/hooks/events.rst

```diff
@@ -192,18 +192,55 @@ Content Authoring Events
      - *Type*
      - *Date added*
 
-   * - `COURSE_CATALOG_INFO_CHANGED <https://github.com/openedx/openedx-events/blob/main/openedx_events/content_authoring/signals.py#L23>`_
+   * - `COURSE_CATALOG_INFO_CHANGED <https://github.com/open-craft/openedx-events/blob/1d017cbf2ab10b99af64ef519f8855063a7abc37/openedx_events/content_authoring/signals.py#L25>`_
      - org.openedx.content_authoring.course.catalog_info.changed.v1
      - `2022-08-24 <https://github.com/openedx/edx-platform/blob/a8598fa1fac5e26ac212aa588e8527e727581742/cms/djangoapps/contentstore/signals/handlers.py#L111>`_
-
-   * - `XBLOCK_PUBLISHED <https://github.com/openedx/openedx-events/blob/main/openedx_events/content_authoring/signals.py#L30>`_
+       
+   * - `XBLOCK_PUBLISHED <https://github.com/open-craft/openedx-events/blob/1d017cbf2ab10b99af64ef519f8855063a7abc37/openedx_events/content_authoring/signals.py#L63>`_
      - org.openedx.content_authoring.xblock.published.v1
      - `2022-12-06 <https://github.com/openedx/edx-platform/blob/master/xmodule/modulestore/mixed.py#L926>`_
 
-   * - `XBLOCK_DELETED <https://github.com/openedx/openedx-events/blob/main/openedx_events/content_authoring/signals.py#L42>`_
+   * - `XBLOCK_DELETED <https://github.com/open-craft/openedx-events/blob/1d017cbf2ab10b99af64ef519f8855063a7abc37/openedx_events/content_authoring/signals.py#L75>`_
      - org.openedx.content_authoring.xblock.deleted.v1
      - `2022-12-06 <https://github.com/openedx/edx-platform/blob/master/xmodule/modulestore/mixed.py#L804>`_
 
-   * - `XBLOCK_DUPLICATED <https://github.com/openedx/openedx-events/blob/main/openedx_events/content_authoring/signals.py#L54>`_
+   * - `XBLOCK_DUPLICATED <https://github.com/open-craft/openedx-events/blob/1d017cbf2ab10b99af64ef519f8855063a7abc37/openedx_events/content_authoring/signals.py#L87>`_
      - org.openedx.content_authoring.xblock.duplicated.v1
      - `2022-12-06 <https://github.com/openedx/edx-platform/blob/master/cms/djangoapps/contentstore/views/item.py#L965>`_
+
+   * - `XBLOCK_CREATED <https://github.com/open-craft/openedx-events/blob/1d017cbf2ab10b99af64ef519f8855063a7abc37/openedx_events/content_authoring/signals.py#L36>`_
```

Comment:
Please update these links to point to openedx/openedx-events instead of pointing to OpenCraft fork.

Reply:
Thank you for pointing out @navinkarkera! I will do it as soon as we get `openedx-events` merged. I will need to squash `openedx-events`, and the commit hash will change again.

ToDo: Update doc links